state/execution_test.go

```diff
@@ -112,15 +55,14 @@ func TestBeginBlockAbsentValidators(t *testing.T) {
 	defer proxyApp.Stop()
 
 	state := state()
-	state.SetLogger(log.TestingLogger())
 
 	// there were 2 validators
-	val1PrivKey := crypto.GenPrivKeyEd25519()
+	/*val1PrivKey := crypto.GenPrivKeyEd25519()
```

Comment:
m?

Reply:
right will look more closely. it turned out we weren't using this, but maybe we should have been